sdk/python/kfp/dsl/executor.py

```diff
@@ -295,9 +296,30 @@ def write_executor_output(self,
         if write_file:
             makedirs_recursively(self.executor_output_path)
             with open(self.executor_output_path, 'w') as f:
+
                 f.write(json.dumps(self.excutor_output))
             return self.executor_output_path
         return None
+    
+    def __check_inf_and_nan_values(self, metadata: object) -> dict:
```

Comment:
should the return type be `object` again to match with the metadata type?

Reply:
That's a good point, I forgot to change that.